frontend/src/routes/(app)/report/[id]/[name]/+layout.server.ts

```diff
@@ -32,6 +34,11 @@ export async function load({ cookies, params, url }) {
 			);
 		}
 	}
+	const [charts, slices, tags] = await Promise.all([
+		zenoClient.getChartsForProjects(reportResponse.report.linkedProjects),
+		zenoClient.getSlicesForProjects(reportResponse.report.linkedProjects),
+		zenoClient.getTagsForProjects(reportResponse.report.linkedProjects)
+	]);
```

Comment:
Should we do this here? Only really relevant if in edit mode, right?

Reply:
Not necessarily, we get the chart from this array when rendering them at all.

Agree it's pretty inefficient though. I think future PR we should redo how we do this querying generally.